Items/04_06_MachineGuns_HeavyWeps_&_RocketLaunchers.txt

```diff
@@ -94,6 +94,32 @@ Movement Speed Penalty: -2m
 * 4 out to 45m
 * 5 out to 80m
 
+==Frontier Flame Thrower FLMR-B $3880 ==

```

Comment:
That's a steep price hike over the level 1 flamethrower. So long as it's worth what you're paying for that's fine.